have error

go-geos@v0.20.3\wkbreader.go:38:130: cannot use _Ctype_ulong(len(wkb)) (value of uint32 type _Ctype_ulong) as _Ctype_size_t value in variable declaration,
wkbreader.go line 38 maybe
return r.context.newGeom(C.GEOSWKBReader_read_r(r.context.cHandle, r.cWKBReader, (*C.uchar)(wkbCBuf), C.size_t(len(wkb))), nil), r.context.err

--- a/wkbreader.go
+++ b/wkbreader.go
@@ -35,7 +35,7 @@ func (r *WKBReader) Read(wkb []byte) (*Geom, error) {
 	wkbCBuf := C.CBytes(wkb)
 	defer C.free(wkbCBuf)
 	r.context.err = nil
-	return r.context.newGeom(C.GEOSWKBReader_read_r(r.context.cHandle, r.cWKBReader, (*C.uchar)(wkbCBuf), C.ulong(len(wkb))), nil), r.context.err
+	return r.context.newGeom(C.GEOSWKBReader_read_r(r.context.cHandle, r.cWKBReader, (*C.uchar)(wkbCBuf), C.size_t(len(wkb))), nil), r.context.err
 }
 
 func (c *Context) destroyWKBReader(cWKBReader *C.struct_GEOSWKBReader_t) {

